types/amplitude-js/index.d.ts

```diff
@@ -103,7 +103,8 @@ export class AmplitudeClient {
     setDeviceId(id: string): void;
     regenerateDeviceId(): void;
 
-    identify(identify_obj: Identify, opt_callback?: Callback): void;
+    identify(identify: Identify, callback?: Callback): void;
+    groupIdentify(groupType: string, groupName: string | string[], identify: Identify, callback?: Callback): void;
```

Comment:
also want to note: technically this method can return an integer, because sometimes the result of pushing to an array is returned: https://github.com/amplitude/Amplitude-JavaScript/blob/main/src/amplitude-client.js#L1175

this is a common pattern for many of the client methods, and i see the definitions here kind of ignore that case, so i followed suit. i assume these methods are intended to only ever return void and the developers of the package probably meant do to
```
array.push(...)
return
```